app/controllers/admin/userManagement.py

```diff
@@ -40,9 +40,26 @@ def manageUsers():
         else:
             removeCeltsStudentStaff(user)
             flash(username+ " is no longer a Celts Student Staff", 'success')
-
     return ("success")
 
+@admin_bp.route('/updateManagers', methods=['POST','GET'])
+def updateProgramManagers():
+    eventData = request.form
+    if  int(eventData['status']) == 0:
+        try:
+            addProgramManager(eventData['userID'],int(eventData['programID']))
+        except:
+            flash('Error while trying to add a manager.')
+    elif int(eventData['status']) == 1:
+        try:
+            removeProgramManager(eventData['userID'],int(eventData['programID']))
+        except:
+            flash('Error while trying to remove a manager.')
+    else:
+        flash('Error while removing a manager.')
+    abort(500)
```

Comment:
You are aborting every time from this controller, so it will always show as a failed request from the user's side.

Reply:
Also, this should only be a POST route. There's no GET requests coming to this route.